### ISSUE TITLE
How can I set up a timer to automatically change to the next screen?

### ISSUE THREAD

**Issue by pcmushthaq** [GursheeshSingh/social_media_widgets#2]
Hi, Thanks for the package . It's working fine. But I'd like to know if I can add a timer so I can automatically change it to next screen after a while

**Comment by GursheeshSingh** [comment]
Not available right now, but this can be added.

https://stackoverflow.com/questions/56540074/how-to-jump-to-another-page-with-pageview-flutter

1. We can ask the user the pass a page controller and user can use that to go to next page
2. Create a custom controller

I prefer custom controller.

I checked your pull request and its looks like you added back button to the example project.
I think the option to add back button should be left to the user and should not be added to the library.

**Comment by pcmushthaq** [comment]
Yeah you are right I'm new to GitHub so I was just trying out these pull requests etc., never mind ..

Thanks I'll look into that

**Comment by GursheeshSingh** [comment]
Welcome!

The back button would added to each screen if I accept the pull request.

Also If we add a timer, we should also add something to show the timer progress like the instagram or whatsapp or snapchat timer.
You can try that and send a pull request

**Comment by aytunch** [comment]
We definitely need a controller for both snapchat circular dismissal widget and Instagram story page view widget. Let's say I have a video playing in Instagram page view and I want it to jump to next video automatically when video is finished, without user swiping. I hope you can embed controllers for these. 
`insta_controller.jumpToIndex(int index)`
`insta_controller.nextPage()`
`insta_controller.previousPage()`

`snapchat_controller.dismiss()`

**Comment by GursheeshSingh** [comment]
Sure I will work on that.

**Comment by denniswhchung** [comment]
I am waiting for the version of "jump to next video" automatically too.
It would be grateful if it can be done asap

**Comment by GursheeshSingh** [comment]
I think it would be better to just have the jumpToNext() and let you control when to jump to next page.

Its almost done.

I haven't worked on the project for a while, so updates have been less.

**Comment by gabofer82** [comment]
@pcmushthaq you can fix this issue? you can share your solution?

**Comment by pcmushthaq** [comment]
> @pcmushthaq you can fix this issue? you can share your solution?

I ended up using my own implementation, I can't find the source code now. I think this package is no longer maintained, you can learn a little bit about PageViews, then it is not that hard to implement.

**Comment by gabofer82** [comment]
> I ended up using my own implementation, I can't find the source code now. I think this package is no longer maintained, you can learn a little bit about PageViews, then it is not that hard to implement.

Thankyou!, Finally I do that....
